### ISSUE TITLE
Unable to change remotect courtesy tone

### ISSUE THREAD

**Issue by G4RKY** [AllStarLink/ASL-Live-Build#36]
Hi,

I've just installed Beta 2.0 release 6 this evening.

In the previous version of Allstarlink, I was able to have different courtesy tones depending on the source of the signal through the repeater, defined in the node stanza of the rpt.conf file. 

For example, if someone called through the locally attached repeater receiver, the system would trigger the unlinked = xx courtesy tone,  as long as there were no other nodes connected via the internet. 

If another remote node was connected, then the remotect = xx courtesy tone plays when the locally attached receiver is the source of the signal. Then, if the signal comes from a remotely connected repeater or any other source coming in via the internet Ie.g. iaxrpt) then the linkunkeyct = xx is used.

With this version, I cannot change the CT that is played by the system when a remote node is connected but the signal comes in via a locally connected receiver via a USB connection.  It is the remotect courtesy tone I cannot change. The others, linkunkeyed and unlinked work as they should.

Even with all the CT definitions tones/morse/files commented out, the same CT is played for remotect. 

The CT in question is a high-low beep-boop noise. No matter what I do, I cannot change or stop it. If there is a remote node connected, whenever the local receiver is used, the following CT is the same beep boop.

I attach my rpt.conf file as text.

[rpt.txt](https://github.com/AllStarLink/ASL-Live-Build/files/9388095/rpt.txt)

Any assistance in resolving this issue would be greatly appreciated.

Many thanks
Matt
G4RKY

**Comment by G4RKY** [comment]
I should add that the system is running on a PC system with a 400Gb HDD and 4Gb of RAM.

**Comment by G4RKY** [comment]
I managed to fix this issue.

In node configuration stanza, the remotect =  setting seems to be ignored. It makes no difference what I set this to!

However, comparing my ASL 1.0 rpt.conf file with the one built during the install of Beta 2.0 rel 6, there is a line missing out of the {Telemetry} stanza for the CT definitions.

If I add a line in there with definition for remotetx =  then all is working fine. I can use this to define the CT played when a locally received signal is passed through the repeater when a remote node is connected. How very confusing!
